test/packageCommonJS.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { Strategy: StrategyNamedImport } = require('../dist/index.cjs');
+const StrategyDefaultImport = require('../dist/index.cjs');
```

Comment:
thought: I don't like this change to be honest (using explicit require/import of `../dist/index.cjs`), since this way we don't get a sanity-check on resolution defined in `package.json`. I'd rather disable the problematic ESlint import rule.

Reply:
I did not change it intentionally, it is result of VSC imports correction after renaming the file. It is actually not in conflict with ESlint import rule. Fixed.